src/main/java/pl/tomaszdziurko/jvm_bloggers/utils/DateTimeUtilities.java

```diff
@@ -22,4 +22,9 @@ public static int daysBetweenDateAndLastFriday(LocalDateTime date) {
         }
     }
 
+    public static LocalDateTime lastPublicationDate(NowProvider nowProvider) {
+        LocalDateTime currentDate = nowProvider.now();
+        int daysSinceLastFriday = daysBetweenDateAndLastFriday(currentDate);
+        return currentDate.minusDays(daysSinceLastFriday).withHour(12).withMinute(0);
+    }
```

Comment:
good idea, we will be able to get rid of passing parameter to lastPublicationDate method like it is now